server/views/partials/layout.njk

```diff
@@ -97,6 +97,11 @@
     initAll()
   </script>
   <script src="/assets/moj/all.js"></script>
+  {% block mojFrontendInit %}
+  <script nonce="{{ cspNonce }}">
+    window.MOJFrontend.initAll()
```

Comment:
We weren't actually initialising the MoJ frontend JavaScript (because we didn't use any components that needed it!) but now we need it for the MoJ sortable table. 

But why is this wrapped in a `{% block mojFrontendInit %}` you may ask? Read my next comment...